js/index.js

```diff
@@ -1,6 +1,27 @@
 const body = document.querySelector('[data-js="body"]');
 const buttonDarkmode = document.querySelector('[data-js="button-darkmode"]');
 
-buttonDarkmode.addEventListener("click", () => {
+buttonDarkmode?.addEventListener("click", () => {
   body.classList.toggle("darkmode");
 });
+
+const answerButton = document.querySelector('[data-js="answer-button"]');
+const answer = document.querySelector('[data-js="hidden-answer"]');
+
+answerButton?.addEventListener("click", () => {
+  answer.classList.toggle("questioncard__answer");
+
+  const buttonInitalText = "Show Answer";
+  if (answerButton.textContent.includes(buttonInitalText)) {
+    answerButton.textContent = "Hide Answer";
+  } else {
+    answerButton.textContent = buttonInitalText;
+  }
+});
+
+const bookmarkButton = document.querySelector('[data-js="bookmark-button"]');
```

Comment:
Maybe I'm missing something, but in your html file you don't have a bookmark button do you? So you don't get an element here. But you don't use the element either, so you might as well remove it unless you do decide to wrap a button around your image.